modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/succubus_milk.dm

```diff
@@ -99,46 +99,43 @@
 		if(mob_breasts.visibility_preference == GENITAL_ALWAYS_SHOW || exposed_mob.is_topless())
 			switch(translation)
 				if(BREAST_SIZE_FLATCHESTED)
```

Comment:
Same thing as with incubus draft, unnecessary return statements in the switch causing inaccessible blocks of code